app/views/planning_applications/review/tasks/_review_assessment.html.erb

```diff
@@ -1,17 +1,19 @@
-<h2 class="app-task-list__section govuk-!-margin-top-8">
-  Review assessment
-</h2>
-<ul class="app-task-list__items" id="review-assessment-section">
-  <li class="app-task-list__item">
-    <span class="app-task-list__task-name">
-      <%= govuk_link_to "Check Community Infrastructure Levy (CIL)", planning_application_review_cil_liability_path(@planning_application) %>
-    </span>
-    <div class="govuk-task-list__status app-task-list__task-tag">
-    <%= render(
-          StatusTags::BaseComponent.new(
-            status: (@planning_application.audits.review_cil_liability.any? ? :complete : :not_started)
-          )
-        ) %>
-    </div>
-  </li>
-</ul>
+<%= bops_task_accordion(id: "review-assessment") do |accordion| %>
+  <% accordion.with_heading(text: "Review assessment") %>
+  <% accordion.with_section(id: "check-cil", expanded: false) do |section| %>
```

Comment:
Can we check for errors to determine whether this section is expanded or not. What happens if a comment is blank when submitting this form?

Reply:
it should also have an error if someone clicks`Save and mark as complete` without selecting Yes or No, right?

Reply:
@EGiataganas I'm not sure. There's currently a test that ensures the assessor can leave it blank, and the reviewer can continue without changing it. So I presume this is intended behaviour.

Reply:
Ok if that's the case, it can be changed at a later stage